lib/TeleIrc.js

```diff
@@ -442,11 +449,22 @@ class TeleIrc {
    * Sends a message to the IRC channel.
    * @param {String} message - The message to send to the IRC channel.
    */
-  sendMessageToIrc(message) {
-    this.ircbot.say(
-      this.config.irc.channel,
-      message
-    );
+  sendMessageToIrc(from, message) {
+    const ircbot = this.ircbot;
+    const messagePrefix = this.config.irc.prefix + from + this.config.irc.suffix + " ";
+    const channel = this.config.irc.channel;
+    const maxLength = this.config.irc.maxMessageLength;
+
+    message.toString().split(/\r?\n/).filter(function(line) {
+      return line.length > 0;
+    }).forEach(function(line) {
+      var linesToSend = ircbot._splitLongLines(line, maxLength - messagePrefix.length, []);
```

Comment:
Instead of using this methodology to split the messages based on the created `maxLength`, it may be useful to look into the `messageSplit` option available through the [irc API](https://node-irc.readthedocs.io/en/latest/API.html#client).

Reply:
I've considered it at first, but then it won't prepend the prefix to the next lines, and that's what we want to avoid :wink: